src/yarpWholeBodyStates.cpp

```diff
@@ -18,6 +18,9 @@
 #include <iCub/skinDynLib/common.h>
 #include <yarp/os/Time.h>
 #include <string>
+#include <boost/concept_check.hpp>
```

Comment:
Spurious? I guess that there is some KDevelop bug inserting this headers from time to time.